modules/cloudwatch_logzio/variables.tf

```diff
@@ -13,6 +13,9 @@ variable "logzio__api_url" {
   default = "https://listener.logz.io:8071"
 }
 
+variable "logzio__api_key" {
```

Comment:
Why switch from SSM parameter to a variable?

This would require putting the api key into the source code. I don't think we typically do that?

Reply:
Well, I suppose you could pass it at the command line, but we also don't typically do that.

Reply:
Actually, I found we do put the datadog API key as a variable: https://github.com/nulogy/terragrunt-modules/blob/4dc53bd9c57ef6144eefabd572a87e7217ffc360/modules/datadog/variables.tf#L5

So there is a precedent

Reply:
Just to be clear, it is still an SSM variable. Just the responsibility of the user of this module to get it from SSM.

Reply:
Ohhhh. Yes, that makes sense.